projects/angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -115,4 +115,8 @@ export const commonStringsDefault: ClrCommonStrings = {
    * Datagrid footer; sr-only text after the number of selected rows.
    */
   selectedRows: 'Selected rows',
+
+  // Accordion/Stepper
+  stepComplete: 'Step {STEP} complete',
+  stepFailed: 'Step {STEP} failed',
```

Comment:
```suggestion
  stepError: 'Error in step {STEP}',
```

I think this would be better.

Reply:
Yes, makes sense. Made the change.

Reply:
Please also rename `stepFailed` to `stepError`.

Reply:
Thanks!